tests/test_util.go

```diff
@@ -57,10 +58,7 @@ func RunTestCase(t *testing.T, testCase *KustomizeTestCase) {
 	}
 
 	fsys := fs.MakeRealFS()
-	// We don't want to enforce the security check.
-	// This is equivalent to running:
-	// kustomize build --load_restrictor none
-	lrc := loader.RestrictionNone
+	lrc := loader.RestrictionRootOnly
```

Comment:
Same here, can we keep this until we have found a tested replacement to use instead of load_restrictor none?

Reply:
Just pushed to commit to reverse this change